Aptoide/aptoide-client-v8
# Feature/asv 915 search ab testing

**What does this PR do?**

This PR aims to implement a generic experiment for search. This means that we retrieve the experiment name from a request to a server so as to give independence to the IoA team to test their new solutions;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] AbTestRepository.java
- [ ] SearchResultPresenter.kt

**How should this be manually tested?**

Do some searches and confirm that, you're assigned to an AB testing group (this should be cached for 24h after first call), that an impression is made when the results show and that an action is made when you click a result that is not an ad;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-915](https://aptoide.atlassian.net/browse/ASV-915)

**Code Review Checklist**
